.github/workflows/docs.yaml

```diff
@@ -59,7 +59,7 @@ jobs:
               uses: stefanzweifel/git-auto-commit-action@v5
               with:
                   commit_message: "chore: Automatic theme updating workflow [skip ci]"
-                  file_pattern: "website/package*.json"
+                  file_pattern: "website/package*.json website/yarn.lock"
```

Comment:
i guess we should do this in all the other repositories like sdks and clients

Reply:
how is this line about npm or yarn? all of them use some lockfile

Reply:
Yep, and the first glob matches everything for npm due to the *, we just needed to add the yarn lockfile here due to us using yarn

Reply:
oh right, so it was only an issue here, got it